modules/aws_eks/aws-eks.json

```diff
@@ -63,6 +63,11 @@
       "default": {},
       "description": "Custom launch template for the underlying ec2s."
     },
+    "ami_type": {
+      "type": "boolean",
+      "description": "The AMI type to use for the nodes.\nFor more information about this, please visit [here](https://docs.aws.amazon.com/eks/latest/APIReference/API_Nodegroup.html#AmazonEKS-Type-Nodegroup-amiType)\nNote: Currently, \"CUSTOM\" ami type is not supported.\n",
```

Comment:
minor: let's keep custom lowercase

Reply:
should we direct them over this page instead?
https://docs.aws.amazon.com/eks/latest/userguide/eks-optimized-amis.html
it has the Bottlerocket AMIs too

Reply:
Too much information and complexity for the User in just one go. If the users want, they can research the supported AMI types on their own. 
This would be much easier IMO, from the user's POV.

Reply:
Let me test whether the lowercase can be directly used or not. What's the reasoning behind keeping it in all lower cases?